iTechArt.Repository/Repositories/GroceryRepository.cs

```diff
@@ -18,15 +18,15 @@ public GroceryRepository(AppDbContext dbContext, IMapper mapper)
             _mapper = mapper;
         }
         /// <summary>
-        /// Get all entities from database
+        /// Get all entities from database.
         /// </summary>
         public async Task<IGrocery[]> GetAllAsync()
         {
             return await _dbContext.Groceries.Select(groceries => _mapper.Map<Grocery>(groceries)).ToArrayAsync();
         }
```

Comment:
inside Select statement, you convert only one grocery, why you wrote 'groceries'?